ui/pages/developer_page.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+class DeveloperPage:
+    def __init__(self, driver):
+        self.driver = driver
+
+    build_on_neon_link_block = (By.XPATH, "//a[contains(.,'Build on Neon')]")
+    developers_url = "https://neonevm.org/developers"
+
+    def click_on_build_on_neon_link_block(self):
+        self.driver.find_element(*DeveloperPage.build_on_neon_link_block).click()
```

Comment:
in selenium, it is necessary to wait for any element before doing anything with it

```suggestion
        from selenium.webdriver.support import expected_conditions as EC
        self.wait.until(EC.presence_of_element_located(**DeveloperPage.build_on_neon_link_block)).click()
```

Reply:
Fixed